tuist/tuist
# Fix `tuist edit` when project contains a Templates folder

Resolves https://github.com/tuist/tuist/issues/4743

### Short description 📝

Templates can import the ProjectDescriptionHelpers target/project. If a Template does this, the generated workspace/project from 'tuist edit' will sometimes not compile because the Templates target may be built before ProjectDescriptionHelpers. This change explicitly adds that dependency in the generated project to ensure ProjectDescriptionHelpers is built before Templates.

### How to test the changes locally 🧐

Create a Tuist project with both `ProjectDescriptionHelpers` and a template where the helpers are imported. Run `tuist edit --permanent`, open the `Manifest.xcworkspace` and make sure the workspace compiles.

### Checklist ✅
